fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -20,17 +20,41 @@
 
 #include "fdbserver/TagPartitionedLogSystem.actor.h"
 
+#include <utility>
+
 #include "flow/actorcompiler.h" // This must be the last #include.
 
-ACTOR Future<Version> minVersionWhenReady(Future<Void> f, std::vector<Future<TLogCommitReply>> replies) {
-	wait(f);
-	Version minVersion = std::numeric_limits<Version>::max();
-	for (auto& reply : replies) {
-		if (reply.isReady() && !reply.isError()) {
-			minVersion = std::min(minVersion, reply.get().version);
+ACTOR Future<Version> minVersionWhenReady(Future<Void> f,
+                                          std::vector<std::pair<UID, Future<TLogCommitReply>>> replies) {
+	try {
+		wait(f);
+		Version minVersion = std::numeric_limits<Version>::max();
+		for (const auto& [_tlogID, reply] : replies) {
+			if (reply.isReady() && !reply.isError()) {
+				minVersion = std::min(minVersion, reply.get().version);
+			}
+		}
+		return minVersion;
+	} catch (Error& err) {
+		if (err.code() == error_code_operation_cancelled) {
+			TraceEvent(SevInfo, "TLogPushCancelled");
```

Comment:
`s/SevInfo/SevWarnAlways/`

Reply:
I think this will cause a lot of joshua tests fail due to the number of warn would blast.

Reply:
I don't think so. For production, we want to this to be `SevWarnAlways` for better visibility, so let's change it for non-simulations.